modules/libvirt/host/cloud_init.cfg

```
@@ -0,0 +1,31 @@
+#cloud-config
+# vim: syntax=yaml
+# set locale
+locale: en_US.UTF-8
+
+# set timezone
+timezone: Europe/Paris
+
+ssh_pwauth: True
+chpasswd:
+  list: |
+     root:linux
+  expire: False
+
+disable_root: false
+
+# NOTE: This is distro specific.
+apt_update: true
```

Is this silently ignored on platforms that do not support this flag?

---

at moment i think that our SLES images are not using `cloudinit`, even if we pass it throug.

To me, i think that when we switch to JEOS/SLES we will need to have 2 different cloudinit files because of the pkg names differs.
Imho this is something that we can postpone and openining an ISSUE as `follow-up` of this

---

not only the pkgs name but i think other things differ so we might have 2 files in case